tests/lib/rules/jsx-closing-tag-location.js

```diff
@@ -95,20 +162,38 @@ ruleTester.run('jsx-closing-tag-location', rule, {
           foo
         </>
       `,
-      errors: [{ messageId: 'matchIndent' }],
+      errors: [{ messageId: 'matchIndent' }], // here
     },
     {
       code: `
-        <>
-          foo</>
+        const x = () => {
+          return <App>
+              foo</App>
+        }
       `,
-      features: ['fragment', 'no-ts-old'], // TODO: FIXME: remove no-ts-old and fix
       output: `
-        <>
-          foo
-        </>
```

Comment:
is there a reason we have to remove these existing tests?

Reply:
I made a mistake. Sorry about that. I'll add them back and commit the changes.